apollo-client/src/main/java/com/ctrip/framework/apollo/spring/config/PropertySourcesProcessor.java

```diff
@@ -62,7 +62,7 @@
 
   private final ConfigPropertySourceFactory configPropertySourceFactory = SpringInjector
       .getInstance(ConfigPropertySourceFactory.class);
-  private final ConfigUtil configUtil = ApolloInjector.getInstance(ConfigUtil.class);
+  private ConfigUtil configUtil;
```

Comment:
Is it necessary to change this logic too? I noticed this line of code was introduced 3 years ago..

Reply:
currently it's ok if we keep the origin logic, but i thinks it's more safe if we will make this change